hawkular-alerts-engine/src/test/java/org/hawkular/commons/cassandra/EmbeddedCassandra.java

```diff
@@ -1,43 +0,0 @@
-/*
```

Comment:
Why this file has been dropped ?
This was placed here for some profiles where we don't use a cassandra from a .war.
Is that being tested ?